openest/generate/smart_curve.py

```diff
@@ -10,6 +10,7 @@
 """
 
 import numpy as np
+import xarray as xr
```

Comment:
I don't think this is used anywhere. Could just remove?

Reply:
Dropped.